### PR TITLE
Fix for issue #1037

### PR DESCRIPTION
**Pull request by mithlesh4257**
Fix for issue #1037
Explicitly show that the license text is a link.

@psh, @misaochan Kindly review it.

### CODE REVIEW

**Comment by maskaravivek** [comment]
@mithlesh4257 Any particular reason for opening a new PR for the same issue? 

IMO the discussions get lost if the same PR is not maintained.

**Comment by mithlesh4257** [comment]
Actually @maskaravivek, I don't know how to manage this huge repository multiple file. What I mean to say I don't know how to add another commit to same PR.

**Comment by maskaravivek** [comment]
After committing your changes locally, just do a git push <branch_name>. It will automatically add that commit to the PR. :)

**Comment by mithlesh4257** [comment]
Thanks @maskaravivek.

File: app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java [hunk 1]
```diff
@@ -36,21 +40,23 @@
...
-import fr.free.nrw.commons.di.CommonsDaggerSupportFragment;
 import fr.free.nrw.commons.settings.Prefs;
+import fr.free.nrw.commons.ui.widget.HtmlTextView;
 import timber.log.Timber;
 
 import static android.view.MotionEvent.ACTION_DOWN;
 import static android.view.MotionEvent.ACTION_UP;
 
-public class SingleUploadFragment extends CommonsDaggerSupportFragment {
+public class SingleUploadFragment extends DaggerFragment {
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: Why did you changed this?

File: app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java [hunk 2]
```diff
@@ -240,9 +246,15 @@ boolean descriptionInfo(View view, MotionEvent motionEvent) {
         }
         return false;
     }
+    /*
+
+
+
+
+    */
```
Comment (neslihanturan) [inline app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: For what this empty lines stands for?

File: app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java (hunk 1 above)
Comment (maskaravivek) [reply app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: Please undo it. Usage of `DaggerFragment` would cause crashes.

Comment (mithlesh4257) [reply app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: Sure I will remove it @maskaravivek

Comment (mithlesh4257) [reply app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: @maskaravivek, I had deleted the previous Commons forked repository due to some bugs and forked again. Can I make another PR as through this I'm unable to change my files?

Comment (mithlesh4257) [reply app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: Should I make another PR that is in that I just have changed "upload/SingleUploadFragment.java" file as I had deleted the previously forked repo, I'm not also able to change this file directly from this PR.
@maskaravivek, @neslihanturan Please help!

Comment (mithlesh4257) [reply app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: Or I have one solution for it, May you edit this @maskaravivek , @neslihanturan?

Comment (neslihanturan) [reply app/src/main/java/fr/free/nrw/commons/upload/SingleUploadFragment.java]: @mithlesh4257 you can fork it again, and make a fresh PR don't worry. I am closing this. Please follow this document before https://github.com/commons-app/apps-android-commons/wiki/Setting-up-dev-enviroment by changing "maskaravivek" username with yours. And please make sure all of the steps we discussed here are covered before preparing PR. Make sure you checked your commits and added meaningful commit messages before pushing.

Besides, if you want to leave this task for someone else to handle, please notify us.

**Pull request closed without merging**